### DISCUSSION TITLE
Packaging for Windows

### DISCUSSION THREAD

**Discussion by fkromer** [strictdoc-project/strictdoc#615] [Ideas]
As soon as [*Windows support: known issues*](https://github.com/strictdoc-project/strictdoc/issues/491) is fixed it would be reasonable to provide `strictdoc`  in a Windows packaging format. Options are [*scoop*](https://scoop.sh/) and [*chocolatey*](https://chocolatey.org/). Until then Windows users may consider [Dockerized strictdoc](https://github.com/strictdoc-project/strictdoc/issues/562) in case one don't want to install `strictdoc` as Python package into the system/user Python installation.

**Comment by stanislaw** [comment]
Moving this to Discussions for now.
